src/fides/ctl/core/utils.py

```diff
@@ -138,6 +141,19 @@ def git_is_dirty(dir_to_check: str = ".") -> bool:
     Checks to see if the local repo has unstaged changes.
     Can also specify a directory to check.
     """
+
+    try:
```

Comment:
@NevilleS is this more what you had in mind?

This won't really be possible to test I think? But I tested it manually in a container without git and this seems to work

Reply:
I tested by changing to the docker directory, then in a python repl ran the following and got the result I expected.

```console
>>> from fides.ctl.core.utils import git_is_dirty
>>> print(git_is_dirty())
No git repo detected at '.git/', skipping git check...
False
```

We could check the parent directory for a `.git` directory, but that could potentially be error prone.